### DISCUSSION TITLE
Feature Request: Add website monitoring/alerting based on client hostnames

### DISCUSSION THREAD

**Discussion by lakesol** [umbraco/Umbraco.Cloud.Issues#813] [Ideas]
### Issue description

We recently had an issue where it became apparent that website monitoring is happening on the umbraco hostname (e.g. <clientname>.euwest01.umbraco.io only and not client specific hostnames <clientname>.com.

This resulting in the site appearing as OK/online to Umbraco but down the public.

Can we suggest that monitoring/alerting is enhanced so that specific hostnames can be added to the monitoring list?

e.g.

![image](https://github.com/umbraco/Umbraco.Cloud.Issues/assets/5919075/689aa0c2-fc9c-4642-8990-be3df81a9d5f)

Consideration should be given where these alerts go in addition to Umbraco support, maybe users can subscribe to alerts.

**Comment by sajumb** [comment]
You're absolutely right. Currently, our monitoring is centered around the Umbraco hostname (e.g. .euwest01.umbraco.io) and doesn't include custom hostnames. We acknowledge that this might be a nice feature to add and will be working towards enhancing our monitoring/alerting capabilities to include specific hostnames in the future. In the meantime, using an uptime tracker for your custom hostnames could be a helpful supplement. Thank you for your feedback.

**Comment by pijemcolu** [comment]
This is now a feature for some time.

https://docs.umbraco.com/umbraco-cloud/optimize-and-maintain-your-site/monitor-and-troubleshoot/hostname-monitoring

Alerting part of the conversation can be followed up on here: https://github.com/umbraco/Umbraco.Cloud.Issues/discussions/1042
